modules/auxiliary/admin/http/epmp1000_ping_cmd_exec.rb

```diff
@@ -83,8 +83,6 @@ def cmd_exec(config_uri, cookie)
     )
 
     if good_response
-      vprint_line("#{res.body}")
```

Comment:
I think this should be changed to print_line.

Reply:
Ditto [here](https://github.com/rapid7/metasploit-framework/pull/9316#discussion_r158294341)